### ISSUE TITLE
Keyboard shortcut request: swithing/moving tabs

### ISSUE THREAD

**Issue by aurelijusrozenas** [TablePlus/TablePlus-Linux#40]
Hi,

it would be great to have these keyboard shortcuts:
1. Ctrl + page up/down - to switch between tabs.
2. Ctrl + Shift + page up/down - to move tab left/right.

This is default behaviour in most browsers and some console/terminal apps I am using and extremely convenient if you do not want to move hand from keyboard to mouse.

Pretty please (ง '̀͜ '́ )ง

**Comment by dvkndn** [comment]
I agree, we will put this into our list :+1: I will update here when there is a new version for this

**Comment by PHLAK** [comment]
Tab navigation shortcuts would be great, but I'd prefer (<kbd>Shift</kbd>+)<kbd>Ctrl</kbd>+<kbd>Tab</kbd> for navigating tabs.

**Comment by geoffyang** [comment]
Is there a reason `Tab` isn't allowed in the keymap customization?
